src/ecmascript_simd.js

```diff
@@ -120,6 +120,12 @@ function simdCreate(type) {
   return type.fn.apply(type.fn, lanes);
 }
 
+function simdCreateFromBuffer(type, buffer) {
+  var newValue = type.fn.apply(type.fn, lanes);
+  newValue.s_ = new type.view(buffer);
```

Comment:
This aliases the underlying buffer, but if it's another SIMD value, won't changes to the new instance affect the original one?